src/Extensions/Abstractions/ExecutionContext/BaseExecutionContext.cs

```diff
@@ -36,10 +42,10 @@ public class BaseExecutionContext : IExecutionContext
 		public BaseExecutionContext(IHostEnvironment? hostEnvironment = null)
```

Comment:
Make sure it's covered by unit tests

Reply:
Updated unit tests for covering additional exposed variable